2023-round-2/tiwavaldese/nx_version.txt

```diff
@@ -0,0 +1 @@
+print(nx.__version__)
```

Comment:
Could you put the output of this command? Thanks!

Reply:
ok i will